src/main/java/ch/njol/skript/expressions/ExprFormatTime.java

```diff
@@ -40,7 +40,7 @@
 import ch.njol.util.Kleenean;
 
 @Name("Formatted time")
-@Description("Converts date to human-readable text format. By default, yyyy-MM-dd HH:mm:ss z will be used. For reference, see this "
+@Description("Converts date to human-readable text format. By default, yyyy-MM-dd HH:mm:ss (e.g. '2018-03-30 16:03:12') will be used. For reference, see this "
```

Comment:
You removed the `z` which is used for timezones, and surrounding the format with ' ' would be good
```suggestion
@Description("Converts date to human-readable text format. By default, 'yyyy-MM-dd HH:mm:ss z' (e.g. '2018-03-30 16:03:12 UTC') will be used. For reference, see this "
```